source/content/guides/edge-integrations/03-drupal-sdk.md

```diff
@@ -22,20 +22,28 @@ The Edge Integrations personalization system for Drupal consists of two main par
 
 ## Before You Begin
 
+1. As per the [Configuration Overview](https://pantheon.io/docs/guides/edge-integrations/configuration-overview/), ensure that your site's vary headers have been provisioned by Professional Services.
+
 1. Ensure you have a working Drupal 8+ installation ready.
 
-1. Install [Smart Content CDN module](https://github.com/pantheon-systems/smart_content_cdn) and its dependencies, which can be pulled with [Composer](https://getcomposer.org/) from their corresponding Pantheon repositories. This includes:
+1. Install [Smart Content CDN module](https://github.com/pantheon-systems/smart_content_cdn) and its dependencies, which can be pulled with [Composer](https://getcomposer.org/) from their corresponding Pantheon repositories.
```

Comment:
The Pantheon Edge Integrations library (`pantheon-systems/pantheon-edge-integrations`) is a dependency of `smart_content_cdn`. This makes it sound like you need to install both, when you should just need to install the `smart_content_cdn` module and it will pull down the global library via Composer.